spec/std/benchmark_spec.cr

```diff
@@ -12,21 +12,22 @@ describe Benchmark::IPS::Job do
   it "works in general / integration test" do
     # test several things to avoid running a benchmark over and over again in
     # the specs
-    j = Benchmark::IPS::Job.new(0.001, 0.001, interactive: false)
-    a = j.report("a") { sleep 0.001 }
-    b = j.report("b") { sleep 0.002 }
-
-    j.execute
-
-    # the mean should be calculated
-    a.mean.should be > 10
-
-    # one of the reports should be normalized to the fastest but do to the
-    # timer precisison sleep 0.001 may not always be faster than 0.002 so we
-    # don't care which
-    first, second = [a.slower, b.slower].sort
-    first.should eq(1)
-    second.should be > 1
+    File.open File::DEVNULL, "w" do |io|
+      j = Benchmark::IPS::Job.new(0.001, 0.001)
+      a = j.report("a") { sleep 0.001 }
+      b = j.report("b") { sleep 0.002 }
+      j.execute interactive: false, io: io
+
+      # the mean should be calculated
+      a.mean.should be > 10
+
+      # one of the reports should be normalized to the fastest but do to the
+      # timer precisison sleep 0.001 may not always be faster than 0.002 so we
+      # don't care which
+      first, second = [a.slower, b.slower].sort
+      first.should eq(1)
+      second.should be > 1
+    end
```

Comment:
Shouldn't the old usage still be supported? If so, why change the test instead of adding a new one?

Reply:
Because we don't want the output to be sent in the STDOUT, which is the default (as before)

Reply:
I can create a new tests

Reply:
I can create new tests